lang/css/css.talon

```diff
@@ -32,8 +32,8 @@ value <user.number_string> point <digit_string> [{user.css_unit}]:
 value <user.text>: user.insert_formatted(text, "DASH_SEPARATED")
 
 variable <user.text>:
-    name = user.formatted_text(text, settings.get("user.code_public_variable_formatter"))
-    insert("var(--{name})")
+    name = user.formatted_text(text, "DASH_SEPARATED")
+    insert("var(--{name});")
```

Comment:
I dunno if the trailing semicolon is what you'd always want. I know people most often use CSS variables for single-value colours, but you could use them in multi-value contexts as well.